src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -543,34 +562,54 @@
      :on-long-press (fn []
                       (on-long-press
-                       (concat [{:on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
+                       (concat [{:type     :main
+                                 :on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
                                  :id       :reply
+                                 :icon     :main-icons/reply-context20
                                  :label    (i18n/label :t/message-reply)}
-                                {:on-press #(re-frame/dispatch [:chat.ui/save-image-to-gallery (:image content)])
+                                {:type     :main
+                                 :on-press #(re-frame/dispatch [:chat.ui/save-image-to-gallery (:image content)])
                                  :id       :save
-                                 :label    (i18n/label :t/save)}
-                                {:on-press #(images/download-image-http
+                                 :icon     :main-icons/save-context20
+                                 :label    (i18n/label :t/save-image-library)}
+                                {:type     :main
+                                 :on-press #(images/download-image-http
                                              (get-in message [:content :image]) preview/share)
                                  :id       :share
-                                 :label    (i18n/label :t/share)}]
+                                 :icon     :main-icons/share-context20
+                                 :label    (i18n/label :t/share-image)}]
                                (when (and outgoing config/delete-message-enabled?)
-                                 [{:on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
-                                   :label    (i18n/label :t/delete)
+                                 [{:type     :danger
+                                   :on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
+                                   :label    (i18n/label :t/delete-for-everyone)
+                                   :icon     :main-icons/delete-context20
                                    :id       :delete}]))))}]
    reaction-picker])
 
 (defmethod ->message constants/content-type-audio []
   (let [show-timestamp? (reagent/atom false)]
-    (fn [{:keys [outgoing] :as message}
+    (fn [{:keys [outgoing pinned] :as message}
          {:keys [on-long-press modal]
           :as   reaction-picker}]
       [message-content-wrapper message
        [react/touchable-highlight
         (when-not modal
           {:on-long-press
            (fn [] (on-long-press (if (and outgoing config/delete-message-enabled?)
-                                   [{:on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
-                                     :label    (i18n/label :t/delete)
+                                   [{:type     :main
```

Comment:
but i believe it won't work? we have a PR with implementation but its not merged yet

Reply:
https://github.com/status-im/status-mobile/pull/13808

Reply:
Ok I tested the options for audio and seems that nothing broke, we can keep them for now unless QA finds something